webapp/src/components/admin_settings/job_table/job_table.tsx

```diff
@@ -82,6 +86,16 @@ export default class JobTable extends React.Component<State, Props> {
 
         return (
             <div className={classNames('JobTable', 'job-table__panel', this.props.className)}>
+                <div className='form-group'>
+                    <label className='control-label col-sm-4'>
+                        {'TSDB Stats:'}
+                    </label>
+                    <div className='col-sm-8'>
+                        <span>
+                            {`Number of samples: ${this.state.stats?.num_samples},\nNumber of series: ${this.state.stats?.num_series}`}
+                        </span>
```

Comment:
While this is better than nothing of course, I am thinking that from a non-technical user perspective, it will be even better to have these fields:
1. Oldest timestamp present.
2. Most recent timestamp.
3. Total size of data in the backing store.

IMO these will really help a sysadmin to gauge what's going on in the system. The oldest timestamp will also verify that the cleanup of older data is working as expected, and the total size will help them take a better call on what is the best duration span that they can hold.

Reply:
We can add them here, those values are now only being used for limit the date picker. Need to think about the visualization though 🤔

Reply:
Sure, we can have them in a separate PR if that's too much work though. I don't want to hold this up.

Reply:
You asked for it, it's been delivered. cc @agnivade 

<img width="907" alt="resim" src="https://github.com/mattermost/mattermost-plugin-metrics/assets/2153367/ac9fcb21-883f-4749-b072-d219687da4d6">